horace_core/sqw/@sqw/calculate_qw_pixels2.m

```diff
@@ -36,8 +36,11 @@
 idet = idx(2,:)';
 ien = idx(3,:)';
 
+remap = containers.Map(unique(irun), 1:numel(win.header));
+irun = arrayfun(@(x) remap(x), irun);
+
 if ~iscell(win.header)
-    header = {win.header};
+    header = num2cell(win.header)';
```

Comment:
To be fair, we should not use `header` in new code. This is legacy method, necessary for writing old format `sqw` files and may be in some printing (also legacy, no time to throw it from there)

This is probably another ticket though.

Reply:
What is the new alternative?

Reply:
`Experiment.IX_experiment` or ` sqw.experiment_info.expdata`